### ISSUE TITLE
Handling of TLD-only domain names

### ISSUE THREAD

**Issue by Sanqui** [ArchiveTeam/ArchiveBot#322]
ArchiveBot doesn't handle archiving http://ai./:
```
ERROR Fetching ‘http://ai/’ encountered an error: Connection closed.
/home/archivebot/.local/lib/python3.5/site-packages/wpull/protocol/http/client.py:185: UserWarning: HTTP session did not complete.
  warnings.warn(_('HTTP session did not complete.'))
ERROR Fetching ‘http://ai/robots.txt’ encountered an error: Connection closed.
ERROR Fetching ‘http://ai/robots.txt’ encountered an error: Connection closed.
ERROR Fetching ‘http://ai/sitemap.xml’ encountered an error: Connection closed.
ERROR Fetching ‘http://ai/’ encountered an error: Connection closed.
  0 bytes.
```

**Comment by JustAnotherArchivist** [comment]
I've confirmed on jap-gavial that ArchiveBot grabs http://ai/ just fine. The problem this morning was that it ran on pipeline:cd8e21a24d32e609e64f1923412a60f7, which has a line `search ovh.net` in its `/etc/resolv.conf`. This caused the job to use ai.ovh.net instead, which resolves (145.239.193.58) but closes the connection immediately.
So in a sense, this is a duplicate of #318. It's a bit of a special case because the domain doesn't contain any dot, which causes the resolver to try the search domain first IIRC. But we shouldn't use a search domain on any ArchiveBot pipelines, and when that is fixed, this issue goes away as well.

**Comment by JustAnotherArchivist** [comment]
Closing as a duplicate of #318.
